docs/source/apis/saas_api.rst

```diff
@@ -0,0 +1,151 @@
+--------------------------
+
+1. After your payment is processed, you will receive an email from us containing your unique API credentials.
+2. Check your inbox (and the spam folder, just in case) for an email from our domain.
+3. Save these credentials securely, as you will need them to access and integrate our API with your applications.
+
+
+Update Billing
+--------------
+
+If you need to update your billing information, fill out the "Update Billing" form.
+
+1. Select "Update Billing" on `Unstructured SaaS API <https://unstructured.io/api-key-hosted>`_ webpage
+2. Upon successful sign-up, you will be redirected to the Stripe payment page.
+3. Follow the instructions to enter your payment details securely.
+4. Confirm the payment to complete this process. Look out for any confirmation prompts or messages that signify a successful transaction.
+
+.. image:: imgs/paid_api/update_billing.png
+  :align: center
+  :alt: Update Billing
+
+
+Opt-in to Marketing and Usage Emails
+------------------------------------
+
+To receive regular updates on usage statistics and marketing emails, please follow these steps:
+
+1. Select "Subscribe to Usage Updates" on `Unstructured SaaS API <https://unstructured.io/api-key-hosted>`_ webpage.
+2. Enter your email address in the provided field.
+3. Click the "Submit" button to complete the subscription process.
+
+You will receive a confirmation email shortly after submitting your information. Please click the confirmation link to complete your opt-in to usage and marketing emails.
+
+.. image:: imgs/paid_api/opt-in.png
+  :align: center
+  :alt: Opt-in Email
+
+
+Rotate a New API Key
+--------------------
+
+If you need to rotate your API key for security reasons or API key regeneration, follow these steps:
+
+1. Select "Get a new API Key" form on `Unstructured SaaS API <https://unstructured.io/api-key-hosted>`_ webpage.
+2. Type your registered *Email* address in the corresponding input field.
+3. Provide your *Current API Key* in the specified field.
+4. Press the *Submit* button to initiate the API key rotation process.
+
+Once you submit the form, our system automatically generates a new API key and deactivates the old one. You will receive the new API key at your registered email address. Be sure to update your applications with the new API key to maintain access to the API services.
+
+.. image:: imgs/paid_api/rotate_key.png
+  :align: center
+  :alt: Rotate API Key
+
+
+Cancel Plan
+-----------
+
+To cancel your current plan, please complete the following steps:
+
+1. Select "Cancel Form" form on `Unstructured SaaS API <https://unstructured.io/api-key-hosted>`_ webpage.
+2. Provide your registered *Email* in the first field.
+3. Enter your *Current API Key* in the following field.
+4. Click the *Submit* button to send your cancellation request.
+
+Upon submission, our team will process your request, and you will receive a confirmation email regarding the cancellation of your plan.
```

Comment:
@treighton and @amanda103 Same question, will the user get an email confirmation after canceling the plan.